plugins/api-docs/src/components/ApiExplorerTable/ApiExplorerTable.tsx

```diff
@@ -39,7 +39,7 @@ const columns: TableColumn<Entity>[] = [
     render: (entity: any) => (
       <Link
         component={RouterLink}
-        to={generatePath(entityRoute.path, {
+        to={generatePath(`../catalog/${entityRoute.path}`, {
```

Comment:
I guess there isn't a better way to do this right now? I imported the route, but that is only a child route and I don't know that the child route is mounted at `/catalog`

Reply:
@Rugvip what's our recommendation at this point?

Reply:
Yeah there's no good way to do this atm. Add a big TODO that this needs to be fixed in the future to avoid this being copied too much.

Reply:
I would interpret your comment in that way that I should continue my work on this PR 😉 A TODO comment sounds good here.